onnxruntime/python/tools/transformers/models/llama/convert_to_onnx.py

```diff
@@ -754,7 +780,12 @@ def main():
 
     location = args.original_model_name if use_auth_token else args.input
 
-    # Use CUDA for LLaMA-2-70B to speed up export and CPU for other models
+    if args.optimize_optimum:
+        config = AutoConfig.from_pretrained(args.original_model_name)
+        optimize_optimum(config, args)
```

Comment:
It appears the config is created from `AutoConfig`. I think we can remove importing `PretrainedConfig` and replace any references to it with `AutoConfig`.